wdl2cwl/tests/wdl_files/tasks/select_all.wdl

```diff
@@ -0,0 +1,11 @@
+version 1.0
+task test_select_all {
+  input {
+    Int one
+    Int? two
+  }
+  command {}
+  output {
+    Array[Int] first1 = select_all([one, two, 1, select_first([two, one])])
+  }
+}
```

Comment:
Not used yet, but just to illustrate for now. Will update the unit test later to use this file (and fix the missing newline at EOF :grimacing: )